test/test-get-passwd.c

```diff
@@ -0,0 +1,60 @@
+#include "uv.h"
```

Comment:
since we have if for all files, please do add it. you can either put your name or "libuv contributors"